src/navigation/index.js

```diff
@@ -3,6 +3,6 @@
  * Sustainable Solutions (NZ) Ltd. 2019
  */
 
-export { Navigator } from './Navigator';
-export { reducer } from './reducer';
+export AppNavigator from './Navigator';
```

Comment:
Think default exports need to be imported and exported to keep the linter happy?

```
import AppNavigator from './Navigator';
import reducer from './reducer';

export { AppNavigator, reducer };
```

Reply:
@joshxg yes, that is how it should be if I was using this `navigation/index.js` reference. Which I am not.

https://github.com/openmsupply/mobile/blob/9cf90124f4aed17eb533fd845572757b68505a06/src/navigation/ReduxNavigation.js#L5

Reply:
My stance is to make the linter happy. If what makes the linter happy should be changed make a new issue to discuss that refactor so we can get this PR going.

Reply:
These exports of AppNavigator and reducer in `src/navigation/index.js` are there but are never used since the import is happening directly (as in the example code I have left above). 

Linter is happy with the entire project, there is no error shown.

Reply:
Resolved with dfa7207